airflow/sensors/sql.py

```diff
@@ -58,39 +62,35 @@ class SqlSensor(BaseSensorOperator):
     ui_color = '#7c7287'
 
     def __init__(
-        self, *, conn_id, sql, parameters=None, success=None, failure=None, fail_on_empty=False, **kwargs
+        self,
+        *,
+        conn_id,
+        sql,
+        parameters=None,
+        success=None,
+        failure=None,
+        fail_on_empty=False,
+        hook_params=None,
```

Comment:
I wonder if we should use `hook_kwargs` for consistency with `Connection.get_hook()`.

Reply:
I definitely agree with you.
But this inconsistency was provided by this PR (https://github.com/apache/airflow/pull/18718), where BaseSqlOperator takes `hook_params` and Connection takes `hook_kwargs`.
If this PR will be merged I'll create another one that resolves it.

Should I create an issue for that?

Reply:
Yes let’s open a new issue and fix this separately. Thankfully we caught this rather quickly so we don’t need to worry about backwards compatibility as long as we can amend it before 2.3 is released!

Reply:
Done
https://github.com/apache/airflow/issues/19638